modules/swagger-codegen/src/main/resources/Java/ApiClient.mustache

```diff
@@ -1,8 +1,9 @@
 package {{invokerPackage}};
 
 import com.fasterxml.jackson.annotation.*;
-import com.fasterxml.jackson.databind.*;
-import com.fasterxml.jackson.datatype.joda.*;
+import com.fasterxml.jackson.databind.*;{{#java8}}
+import com.fasterxml.jackson.datatype.jsr310.*;{{/java8}}{{^java8}}
+import com.fasterxml.jackson.datatype.joda.*;{{/java8}}
```

Comment:
The latest version of jmustache supports newline trimming: https://github.com/samskivert/jmustache#newline-trimming so that you can do something as follows instead:
```
{{#java8}}
import com.fasterxml.jackson.datatype.jsr310.*;
{{/java8}}
{{^java8}}
import com.fasterxml.jackson.datatype.joda.*;
{{/java8}}
```
More of an FYI so that you can use this new feature in your next PR.